t/cobrand/fixmystreet.t

```diff
@@ -355,15 +366,45 @@ FixMyStreet::override_config {
         $mech->create_contact_ok(body_id => $hampshire->id, category => 'Trees', email => 'foo@bexley');
         $mech->create_contact_ok(body_id => $he->id, category => 'Litter (NH)', email => 'litter@he', group => 'National Highways');
         $mech->create_contact_ok(body_id => $he->id, category => 'Potholes (NH)', email => 'potholes@he', group => 'National Highways');
-        my $he_mod = Test::MockModule->new('FixMyStreet::Cobrand::HighwaysEngland');
+
+        our $he_mod = Test::MockModule->new('FixMyStreet::Cobrand::UKCouncils');
```

Comment:
Nothing was being mocked, so the live tilma was being queried by this test. Mocked it out with a variety of options